.github/pull_request_template.md

```diff
@@ -9,5 +9,5 @@
 
 - [ ] Code is formatted and linted (run `pre-commit run --all-files`)
 - [ ] Tests pass (run `make test`)
-- [ ] Documentation has been updated to reflect changes, if applicable, and docs build successfully (run `make docs`)
+- [ ] Documentation has been updated to reflect changes, if applicable
```

Comment:
Is there a reason why the `make docs` instruction is removed?

Reply:
We don't have a dockerfile set up to run the docs so it is inoperable. I created an issue for this.